### ISSUE TITLE
Tikz overlay breaks footer

### ISSUE THREAD

**Issue by josephwright** [josephwright/beamer#278]
Originally reported by: **thomas001le (Bitbucket: [thomas001le](https://bitbucket.org/thomas001le), GitHub: Unknown)**

----------------------------------------

Hello, the following example produces a wrong result:

```
#!latex

\documentclass{beamer}

\usepackage{tikz}

\usetheme{Madrid}

\begin{document}

\begin{frame}% [fragile] -- doesn't help

  Frame 1
  \pause

  Frame 2
  \begin{tikzpicture}[overlay]
    \onslide<1->
  \end{tikzpicture}
  \pause

  Frame 3
\end{frame}

\end{document}
```

In the first frame, bevor the overlay is shown, the footer is missing. The other 2 slides render correctly. The PDF and images showing my result can be found [here](https://www.dropbox.com/sh/u5b7pzfvq9jmr7s/UxDdfjpmEX#/)
I tried 3 PDF viewers, so i really do think this is a problem within beamer. I attached the compilation log. The problem occurs with the most recent texlive version and with the current bitbucket head.

----------------------------------------
- Bitbucket: https://bitbucket.org/rivanvx/beamer/issue/278

**Comment by josephwright** [comment]
*Original comment by* **thomas001le (Bitbucket: [thomas001le](https://bitbucket.org/thomas001le), GitHub: Unknown)**:

----------------------------------------

1. Note: The \onslide<1-> was put in there as some stackoverflow posts suggested this could solve the issue, which does not work here. the error does also occur without the \onslide<1->

2. Note: Tracked it down to [39bc573](https://bitbucket.org/rivanvx/beamer/commits/39bc573c5bfc03fbd23ea3c16c420abfb6cf3d9b). this commit introduces the problem.

**Comment by josephwright** [comment]
*Original comment by* **Joseph Wright (Bitbucket: [josephwright](https://bitbucket.org/josephwright), GitHub: [josephwright](https://github.com/josephwright))**:

----------------------------------------

I see no difference before and after the commit you cite: could you send me PDFs showing the effect (or attach them)?

**Comment by josephwright** [comment]
*Original comment by* **Joseph Wright (Bitbucket: [josephwright](https://bitbucket.org/josephwright), GitHub: [josephwright](https://github.com/josephwright))**:

----------------------------------------

Ah, I see what you are getting at: if you remove the `\onslide<1->` business then there is a difference in behaviour. I'll have to look at how TikZ implements `overlay`, I guess.

**Comment by josephwright** [comment]
*Original comment by* **Joseph Wright (Bitbucket: [josephwright](https://bitbucket.org/josephwright), GitHub: [josephwright](https://github.com/josephwright))**:

----------------------------------------

No, not an overlay issue: something else: investigating.

**Comment by josephwright** [comment]
*Original comment by* **thomas001le (Bitbucket: [thomas001le](https://bitbucket.org/thomas001le), GitHub: Unknown)**:

----------------------------------------

Build with 39bc573, not footer on first slide. I get this result whether i include \onslide<1-> or not. Also i get the same result with the most recent revision of beamer.

**Comment by josephwright** [comment]
*Original comment by* **thomas001le (Bitbucket: [thomas001le](https://bitbucket.org/thomas001le), GitHub: Unknown)**:

----------------------------------------

Last working revision build.

**Comment by josephwright** [comment]
*Original comment by* **Joseph Wright (Bitbucket: [josephwright](https://bitbucket.org/josephwright), GitHub: [josephwright](https://github.com/josephwright))**:

----------------------------------------

Reverse allowing \pause in TikZ pictures

This reopens #275 but fixes #278 which is more important (introduce
no new issues).
